README.md

```diff
@@ -13,7 +13,16 @@ Bevy is a refreshingly simple data-driven game engine built in Rust. It is free
 
 ## WARNING
 
-Bevy is still in the _very_ early stages of development. APIs can and will change (now is the time to make suggestions!). Important features are missing. Documentation is sparse. Please don't build any serious projects in Bevy unless you are prepared to be broken by API changes constantly.
+Bevy is still in the early stages of development.
+APIs can and will change in both trivial and critical ways.
+Important features are incomplete.
+Documentation can be spotty.
```

Comment:
I also don't quite like this sentiment here. It's also "disavowing responsibility" to improve the docs by using the passive voice, which isn't great.

Perhaps "Our documentation is still maturing - individual APIs are generally well documented, but higher level overviews are difficult to find". Again, that's not quite right.

Maybe we can also mention here that assistance can be had on discord or in GitHub discussions?

OTOH I don't want this section to get long - I don't want people's eyes to glaze over reading our readme.

Reply:
The status of documentation depends on the viewpoint. If we're just talking about API documentation, the situation is not that bad, even if it needs significative additions. If we're talking about the Book instead, the situation is such that people _need_ to resort to third-party sources.